domain/src/main/java/com/globant/domain/usecase/implementation/GetAllCharactersFromDatabaseUseCaseImpl.kt

```diff
@@ -0,0 +1,11 @@
+package com.globant.domain.usecase.implementation
+
+import com.globant.domain.entity.Character
+import com.globant.domain.util.Result
+import com.globant.domain.repository.CharacterRepository
+import com.globant.domain.usecase.GetAllCharactersFromDatabaseUseCase
+
+class GetAllCharactersFromDatabaseUseCaseImpl(private val characterRepository: CharacterRepository) :
+    GetAllCharactersFromDatabaseUseCase {
```

Comment:
the code style should be 

class GetAllCharactersFromDatabaseUseCaseImpl(
          private val characterRepository: CharacterRepository
)  :   GetAllCharactersFromDatabaseUseCase {

if it cant fint in one line

Reply:
this is repeated in many files

Reply:
added that code style, thanks